WebRoot/js/ajax/dwt/xforms/XFormItem.js

```diff
@@ -994,14 +994,14 @@ XFormItem.prototype.handleKeyPressDelay = function(ev, domItem) {
 };
 
 XFormItem.prototype.getKeyPressHandlerHTML = function () {
-	// if AjxEnv.isNav, etc.
-	var keydownEv = DwtEvent.ONKEYPRESS;
+  // if AjxEnv.isNav, etc.
+  var keydownEv = DwtEvent.ONKEYPRESS; // "onkeypress"
```

Comment:
i see only some spacing changes and addition of comments

are we fixing something here?